milatools/cli/commands.py

```diff
@@ -81,478 +95,778 @@ def main():
+
+    # ----- mila serve lab ------
+
+    serve_lab_parser = serve_subparsers.add_parser(
+        "lab",
+        help="Start a Jupyterlab server.",
+        formatter_class=SortingHelpFormatter,
+    )
+    serve_lab_parser.add_argument(
+        "PATH",
+        default=None,
+        nargs="?",
+        help="Path to open on the remote machine",
+    )
+    _add_standard_server_args(serve_lab_parser)
+    serve_lab_parser.set_defaults(function=lab)
+
+    # ----- mila serve notebook ------
+
+    serve_notebook_parser = serve_subparsers.add_parser(
+        "notebook",
+        help="Start a Jupyter Notebook server.",
+        formatter_class=SortingHelpFormatter,
+    )
+    serve_notebook_parser.add_argument(
+        "PATH",
+        default=None,
+        nargs="?",
+        help="Path to open on the remote machine",
+    )
+    _add_standard_server_args(serve_notebook_parser)
+    serve_notebook_parser.set_defaults(function=notebook)
+
+    # ----- mila serve tensorboard ------
+
+    serve_tensorboard_parser = serve_subparsers.add_parser(
+        "tensorboard",
+        help="Start a Tensorboard server.",
+        formatter_class=SortingHelpFormatter,
+    )
+    serve_tensorboard_parser.add_argument(
+        "LOGDIR", type=str, help="Path to the experiment logs"
+    )
+    _add_standard_server_args(serve_tensorboard_parser)
+    serve_notebook_parser.set_defaults(function=tensorboard)
+
+    # ----- mila serve mlflow ------
+
+    serve_mlflow_parser = serve_subparsers.add_parser(
+        "mlflow",
+        help="Start an MLFlow server.",
+        formatter_class=SortingHelpFormatter,
+    )
+    serve_mlflow_parser.add_argument(
+        "LOGDIR", type=str, help="Path to the experiment logs"
+    )
+    _add_standard_server_args(serve_mlflow_parser)
+    serve_mlflow_parser.set_defaults(function=mlflow)
+
+    # ----- mila serve aim ------
+
+    serve_aim_parser = serve_subparsers.add_parser(
+        "aim",
+        help="Start an AIM server.",
+        formatter_class=SortingHelpFormatter,
+    )
+    serve_aim_parser.add_argument(
+        "LOGDIR", type=str, help="Path to the experiment logs"
+    )
+    _add_standard_server_args(serve_aim_parser)
+    serve_aim_parser.set_defaults(function=aim)
+
+    args = parser.parse_args()
+    args_dict = vars(args)
+    function = args_dict.pop("function")
+    _ = args_dict.pop("<command>")
+    _ = args_dict.pop("<serve_subcommand>", None)
+    # replace SEARCH -> "search", REMOTE -> "remote", etc.
+    args_dict = _convert_uppercase_keys_to_lowercase(args_dict)
+    assert callable(function)
+    return function(**args_dict)
+
+
+def _convert_uppercase_keys_to_lowercase(args_dict: dict[str, Any]) -> dict[str, Any]:
+    return {(k.lower() if k.isupper() else k): v for k, v in args_dict.items()}
+
+
+def docs(search: Sequence[str]) -> None:
+    url = "https://docs.mila.quebec"
+    terms = "+".join(search)
+    url = f"{url}/search.html?q={terms}"
```

Comment:
```suggestion
    if search:
        terms = "+".join(search)
        url = f"{url}/search.html?q={terms}"
```

Otherwise, when you do not put search terms, it leads to this uninteresting page: https://docs.mila.quebec/search.html?q=